plugins/auth-node/src/types.ts

```diff
@@ -163,6 +163,15 @@ export type AuthResolverContext = {
   ): Promise<BackstageSignInResult>;
 };
 
+/**
+ * Resolver interface for resolving the ownership entity references for entity
+ *
+ * @public
+ */
+export interface AuthOwnershipResolver {
+  getOwnershipEntityRefs(entity: Entity): Promise<string[]>;
```

Comment:
Let's have the return value be `Promise<{ ownershipEntityRefs: string[] }>`, it lets us add things more easily in the future.

Naming-wise I think it would make sense to call this `resolveOwnershipEntityRefs` too?